components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -66,21 +67,27 @@ - (instancetype)initWithPresentedViewController:(UIViewController *)presentedVie
 
     _trackingView = [[MDCDialogShadowedView alloc] init];
 
+#if TARGET_OS_IOS
     [self registerKeyboardNotifications];
```

Comment:
probably, you can also type on your phone. You get a notification on the phone when the cursor is over a textfield.

but the onscreen keyboard overlay appears over the screen and not under it like on iOS